.changeset/pre.json

```diff
@@ -0,0 +1,48 @@
+{
+  "mode": "pre",
+  "tag": "rc",
+  "initialVersions": {
+    "@atproto/api": "0.12.25",
+    "@atproto/aws": "0.2.1",
+    "@atproto/bsky": "0.0.69",
+    "@atproto/bsync": "0.0.5",
+    "@atproto/common": "0.4.1",
+    "@atproto/common-web": "0.3.0",
+    "@atproto/crypto": "0.4.0",
+    "@atproto/dev-env": "0.3.33",
+    "@atproto/did": "0.1.0",
+    "@atproto/identity": "0.4.0",
+    "@atproto-labs/did-resolver": "0.1.1",
+    "@atproto-labs/fetch": "0.1.0",
+    "@atproto-labs/fetch-node": "0.1.0",
+    "@atproto-labs/handle-resolver": "0.1.1",
+    "@atproto-labs/handle-resolver-node": "0.1.1",
+    "@atproto-labs/identity-resolver": "0.1.1",
+    "@atproto-labs/pipe": "0.1.0",
+    "@atproto-labs/rollup-plugin-bundle-manifest": "0.1.0",
+    "@atproto-labs/simple-store": "0.1.1",
+    "@atproto-labs/simple-store-memory": "0.1.1",
+    "@atproto/lex-cli": "0.4.0",
+    "@atproto/lexicon": "0.4.0",
+    "@atproto/jwk": "0.1.1",
+    "@atproto/jwk-jose": "0.1.1",
+    "@atproto/jwk-webcrypto": "0.1.1",
+    "@atproto/oauth-client": "0.1.1",
+    "@atproto/oauth-client-browser": "0.1.1",
+    "@atproto/oauth-client-node": "0.0.1",
+    "@atproto/oauth-provider": "0.1.1",
+    "@atproto/oauth-types": "0.1.1",
+    "@atproto/ozone": "0.1.31",
+    "@atproto/pds": "0.4.42",
+    "@atproto/repo": "0.4.1",
+    "@atproto/syntax": "0.3.0",
+    "@atproto/xrpc": "0.5.0",
+    "@atproto/xrpc-server": "0.5.3"
+  },
+  "changesets": [
+    "brown-dryers-fail",
+    "honest-hornets-approve",
+    "purple-pans-shout",
+    "rare-feet-happen"
+  ]
+}
```

Comment:
Should this file be remove since I also removed every other mention to rc* versions (in both CHANGELOG.md and package.json) ?

Reply:
Yeah, that should be fine—I am maintaining a branch now for future RCs.